packages/theming/win32-theme/src/createOfficeTheme.ts

```diff
@@ -20,20 +20,20 @@ function handlePaletteCall(palette: OfficePalette | CxxException): OfficePalette
  */
 export function createOfficeTheme(paletteName?: string): ThemeReference {
   const [module, emitter] = getThemingModule();
-  const ref = { module, emitter, defaultPalette: module.initialHostThemeSetting || '' };
+  const ref = { module, emitter, themeName: module.initialHostThemeSetting || '' };
 
   const themeRef = new ThemeReference(defaultFluentTheme, () => {
-    const name = paletteName || ref.defaultPalette;
+    const name = paletteName || '';
     const palette = handlePaletteCall(ref.module.getPalette(name));
-    return createPartialOfficeTheme(module, palette);
+    return createPartialOfficeTheme(module, ref.themeName, palette);
   });
 
   // set up the callback for theme changes on the native side
   const onPlatformDefaultsChanged = (args: PlatformDefaultsChangedArgs) => {
-    ref.defaultPalette = (args && args.hostThemeSetting) || ref.defaultPalette;
+    ref.themeName = (args && args.hostThemeName) || ref.themeName;
```

Comment:
Same here, the args object's shape is defined by the native side.

Reply:
Good catch